carapp/__init__.py

```diff
@@ -34,4 +39,7 @@
 manager.add_command("db", MigrateCommand)
 
 
-from carapp import endpoints
+login_manager = LoginManager()
+login_manager.init_app(app)
+
+from carapp import endpoints, authent
```

Comment:
Are you hacking the cycle import issues?

Reply:
yes, let's call it like this, sounds cooler than "prevent to stuck in a cycle" :)